tck/rest/src/main/java/org/eclipse/microprofile/metrics/test/MetricAppBean.java

```diff
@@ -81,6 +81,11 @@ public void countMeA() {
 
     }
 
+    @Counted(name = "metricTest.test1.countMeB", monotonic = true, absolute = true, unit = "jellybean")
```

Comment:
monotonic = true will need to be removed once https://github.com/eclipse/microprofile-metrics/pull/309 is in